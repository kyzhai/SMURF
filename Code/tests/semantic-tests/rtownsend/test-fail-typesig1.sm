/* Multiple identical type signatures are not allowed in the same scope */

a :: Int
a :: a

main = []
