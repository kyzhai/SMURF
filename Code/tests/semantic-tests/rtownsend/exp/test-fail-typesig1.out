Fatal error: exception Sast.Multiple_type_sigs("a")
